File: UltimateTicTacToe/Models/DTOs/BoardGameDTO.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Linq;
using System.Collections.Generic;
using UltimateTicTacToe.Models.Game;
using UltimateTicTacToe.Models.Game.Players;

namespace UltimateTicTacToe.Models.DTOs
{
    public class BoardGameDTO
    {
        public List<List<JObject>> game;
        public PlayerColour? Winner;
        public List<JObject> players;
        public JObject cur;
        public Move lastMove;
        public List<Move> availableMoves;
        public double lastMoveRating;
    }
}

File: UltimateTicTacToe/ClientApp/src/app/models/DTOs/BoardGameDTO.ts
import { Player } from "../player/player.model";
import { Move } from "../move/move.model";
import { BoardGame } from "../boardGame/boardgame/boardgame.model";
import { PlayerColour } from "../player/player.colour.enum";

export class BoardGameDTO {
    public game: Array<Array<BoardGame>>;
    public winner: PlayerColour;
    public cur: Player;
    public players: Array<Player>;
    public lastMove: Move;
    public availableMoves: Array<Move>;
    public lastMoveRating: number;
    public highOption: number;
    public lowOption: number;
}
